src/app/components/client/ExposureCard.tsx

```diff
@@ -18,15 +18,15 @@ import {
 } from "../server/Icons";
 import Image from "next/image";
 import { Button } from "../server/Button";
+import { ExposureType, ExposureTypeEl } from "../server/ExposureType";
 
-export type Props = {
+export type ExposureCardProps = {
   exposureImg: StaticImageData;
   exposureName: string;
-  exposureType: string;
+  exposureType: ExposureType;
```

Comment:
Is this to distinguish between data breaches and data brokers? If so, we probably want to take an `exposureData` element of type `ScanResult | Breach` from [here](https://github.com/mozilla/blurts-server/blob/75e7d70988ddc122e2882feaa69aa807d0218819/src/external/onerep.js#L249-L264) and [here](https://github.com/mozilla/blurts-server/blob/75e7d70988ddc122e2882feaa69aa807d0218819/src/app/(nextjs_migration)/(authenticated)/user/breaches/breaches.d.ts#L65-L86), and write a [type guard function](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) that distinguishes between the two.

And possibly then pass the result on to breach- and data broker-specific components, so they can make assumptions about what properties are available?